app/poros/merchant.rb

```diff
@@ -0,0 +1,17 @@
+class Merchant 
+  attr_reader :id, :name#, :items
+
+  def initialize(data)
+    @id = data[:id]
+    @name = data[:attributes][:name]
+    # @items = merchant_items
+  end
+
+  # def merchant_items
+  #   items_data = MerchantService.merchant_items(id)[:data]
+
+  #   items_data.map do |data|
+  #     Item.new(data)
+  #   end
+  # end
```

Comment:
The approach you chose (a method on `MerchantFacade` that returns a merchant's items) works really well too, but what I like about this commented out code is that you theoretically wouldn't need to set up as many instance variables in `MerchantsController`.